crates/rpc-client/src/utils.rs

```diff
@@ -30,7 +30,7 @@ impl TracingHttpClient {
         self.inner.url()
     }
 
-    #[instrument(target = "gw-rpc-client", skip_all, err, fields(method, params = field::Empty))]
+    #[instrument(target = "gw-rpc-client", skip_all, fields(method, params = field::Empty))]
```

Comment:
Removed the error from trace because displaying the error without error chain is not very useful.